### PR TITLE
Sorted documentsby created_at and added date field to documents page

### PR DESCRIPTION
**Pull request by 252afh**
## Context

Documents should be sorted so most recent is first, and display the created datetime to the user

## Changes proposed in this pull request

- Added jinja2 filter for converting a datetime to localtime
- Added column to documents page to show created datetime

## Guidance to review

Add some documents and check the datetime shown on the documents page

## Things to check

- [ ] I have added any new ENV vars in all deployed environments
- [ ] I have tested any code added or changed
- [ ] I have run integration tests

_Closing until Kevin can work with me on fixing the table layout on mobile_

### CODE REVIEW

**Comment by KevinEtchells** [comment]
Just to confirm the table layout on mobile is now fixed :-)

File: django_app/redbox_app/jinja2.py [hunk 1]
```diff
@@ -47,6 +49,13 @@ def humanize_timedelta(minutes=0, hours_limit=200, too_large_msg=""):
         return humanize.precisedelta(delta, minimum_unit="minutes")
 
 
+def to_user_timezone(value):
```
Comment (brunns) [inline django_app/redbox_app/jinja2.py:52]: So, we are storing UTC in the database, and just converting to user TZ in the template? Nice.

**Review by brunns — APPROVED** [review]
Looks good.

**Comment by brunns** [comment]
Probably worth merging main in again and running `make format lint`.

**Pull request merged**